lib/taskana-core/src/main/java/pro/taskana/task/api/TaskQuery.java

```diff
@@ -941,20 +941,20 @@ public interface TaskQuery extends BaseQuery<TaskSummary, TaskQueryColumnName> {
    * compared in SQL with the LIKE operator. You may use a wildcard like % to specify the pattern.
    * If you specify multiple arguments they are combined with the OR keyword.
    *
-   * @param company the company of your primary object reference
+   * @param companies the companies of your primary object reference
    * @return the query
    */
-  TaskQuery primaryObjectReferenceCompanyLike(String... company);
+  TaskQuery primaryObjectReferenceCompanyLike(String... companies);
```

Comment:
did't we agree to fully name the paramter?
In this example `primaryObjectReferenceCompanies` instead of `companies`

Reply:
I leave it like this as discussed :)